kornia/image/image.py

```diff
@@ -267,3 +278,18 @@ def write(self, file_path: str | Path) -> None:
         if self.channels_order == ChannelsOrder.CHANNELS_LAST:
             data = data.permute(2, 0, 1)
         write_image(file_path, data)
+
+    def print(self, max_width: int = 256) -> None:
```

Comment:
Probably add `os.get_terminal_size().columns` as the default?

Reply:
mmm, tried and not working very well